integration-tests/src/test/java/oracle/kubernetes/operator/ItUsabilityOperatorHelmChart.java

```diff
@@ -316,53 +318,38 @@ public void testSecondOpSharingSameTargetDomainsNsNegativeInstall() throws Excep
     Operator secondoperator = new Operator(operatorMap, true, true, false, RestCertType.SELF_SIGNED);
     String oprelease = (String)(secondoperator.getOperatorMap()).get("releaseName");
     String opnamespace = (String)(secondoperator.getOperatorMap()).get("namespace");
+    Exception installException = null;
+    Exception infoException = null;
     try {
       secondoperator.callHelmInstall();
-      throw new RuntimeException(
-          "FAILURE: Helm installs second operator with same as first operator's target domains namespaces ");
+      installException = new Exception(
+          "FAILURE: Second operator Helm Installation MUST fails with common domain target namespace");
 
     } catch (Exception ex) {
       LoggerHelper.getLocal().log(Level.INFO, "Caught exception " + ex.getMessage() + ex.getStackTrace());
       if (!ex.getMessage()
-          .contains(
-              "Error: release "
-                  + oprelease
-                  + " failed: rolebindings.rbac.authorization.k8s.io "
-                  + "\"weblogic-operator-rolebinding-namespace\" already exists")) {
-        throw new RuntimeException(
-            "FAILURE: Helm installs second operator with same as first operator's "
-                + "target domains namespaces does not report expected message "
+          .contains("weblogic-operator-rolebinding-namespace")) {
+        infoException = new Exception(
+            "Exception does not report expected message "
                 + ex.getMessage());
       }
-      String cmdLb = "";
-      if (BaseTest.HELM_VERSION.equals("V2")) {
-        cmdLb = "helm list --failed " + "  | grep " + oprelease;
-      }
-      if (BaseTest.HELM_VERSION.equals("V3")) {
-        cmdLb = "helm list --failed --namespace " + opnamespace + "  | grep " + oprelease;
-      }
-      LoggerHelper.getLocal().log(Level.INFO, "Executing cmd " + cmdLb);
-      ExecResult result = ExecCommand.exec(cmdLb);
-      if (result.exitValue() != 0) {
-        throw new RuntimeException(
-            "FAILURE: Helm installs second operator with same as first operator's target domains namespaces ");
-      }
-
     } finally {
       if (firstoperator != null) {
         firstoperator.destroy();
       }
-      if (secondoperator != null) {
-        secondoperator.destroy();
-      }
     }
+    Assertions.assertNull(installException,"Second operator Helm Installation MUST fail");
```

Comment:
space after comma
installation should not have capital

Reply:
done